src/notional/orm.py

```diff
@@ -174,9 +174,13 @@ class ConnectedPage:
     All changes are committed in real time.
     """
 
-    def __init__(self, **data):
-        """Construct a page from the given data dictionary."""
-        self._notional__page = Page(**data) if data else None
+    def __init__(self, page: Page):
```

Comment:
⚠️ BREAKING CHANGE - however, this should mostly be used internally at this point.